### PR TITLE
chore: Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by lumaxis**
I keep having to remove these placeholders and the `>` for quotes and `*` for bulleted lists don't actually feel fitting for me. Also, imo, it's pretty clear that a PR description is used to describe what a PR is about. Replacing the "What / Why" heading and leaving it as a comment, to encourage people to actually fill it out.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# What / Why
-<!-- Describe the request in detail -->
-> n/a
+<!-- What / Why >
```
Comment (darcyclarke) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Is the `>` meant to be a closing comment tag? (ie. `-->`)

Comment (lumaxis) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Oh, yes! Thanks 🙂

**Comment by gordey4doronin** [comment]
1. I personally like bulleted lists. Can we keep them? Or can I continue using them?
2. Not sure what's wrong with `What / Why` header. I am personally okay with that.
3. Definitely 👍 for removing `> n/a`.

Screenshot 2020-07-22 at 14 00 08
Screenshot 2020-07-22 at 14 00 13
Screenshot 2020-07-22 at 14 00 28

**Comment by gordey4doronin** [comment]
BTW Does it also affect our public repos and people contributing to them?

**Comment by lumaxis** [comment]
@gordey4doronin Continue using bulleted lists as you see fit 😄

**Pull request merged**